orchagent/portsorch.cpp

```diff
@@ -1882,7 +1881,6 @@ bool PortsOrch::initPort(const string &alias, const int index, const set<int> &l
 
                 /* Add port to port list */
                 m_portList[alias] = p;
-                m_port_ref_count[alias] = 0;
```

Comment:
i think you accidently remove this line. Can you put it back?

Reply:
this line was somehow showing as a new addition as seen in your pervious comment. I just  re-added this line.